indico/modules/events/abstracts/views.py

```diff
@@ -28,11 +28,13 @@ class WPManageAbstracts(WPJinjaMixin, WPConferenceModifBase):
     def getJSFiles(self):
         return (WPConferenceModifBase.getJSFiles(self) +
                 self._asset_env['markdown_js'].urls() +
+                self._asset_env['selectize_js'].urls() +
```

Comment:
I still hope to move away from separate js bundles in favor of one big indico.js/indico.css bundle for 2.0 so maybe it's something we can keep in mind for then.